packages/kbn-optimizer/limits.yml

```diff
@@ -58,7 +58,7 @@ pageLoadAssetSize:
   telemetry: 51957
   telemetryManagementSection: 38586
   transform: 41007
-  triggersActionsUi: 105600 # This is temporary, check https://github.com/elastic/kibana/pull/130710#issuecomment-1119843458 & https://github.com/elastic/kibana/issues/130728
+  triggersActionsUi: 105800 # https://github.com/elastic/kibana/pull/130710#issuecomment-1119843458 & https://github.com/elastic/kibana/issues/130728
```

Comment:
No longer temporary?

Reply:
I love the `This is temporary` comment, feels like a nice little commitment and keeps it in people's faces every time that number is updated without those issues being fixed.